core/src/main/java/org/apache/iceberg/encryption/StandardKeyMetadata.java

```diff
@@ -31,7 +31,7 @@
 import org.apache.iceberg.relocated.com.google.common.collect.ImmutableMap;
 import org.apache.iceberg.types.Types;
 
-class StandardKeyMetadata implements EncryptionKeyMetadata, IndexedRecord {
+public class StandardKeyMetadata implements EncryptionKeyMetadata, IndexedRecord {
```

Comment:
We can't make this class public because it implements an Avro interface. If it needs to be public, we will have to extract `StandardKeyMetadata` as an interface.